crates/binstalk-fetchers/src/quickinstall.rs

```diff
@@ -95,13 +97,19 @@ impl super::Fetcher for QuickInstall {
             stats_url: Url::parse(&format!("{STATS_URL}/{package}.tar.gz",))
                 .expect("stats_url is pre-generated and should never be invalid url"),
             package,
+            signature_policy,
 
             target_data,
         })
     }
 
     fn find(self: Arc<Self>) -> JoinHandle<Result<bool, FetchError>> {
         tokio::spawn(async move {
+            // until quickinstall supports signatures, blanket deny:
```

Comment:
I would like to discuss on what signing strategy should quickinstall support.

I personally think quickinstall should use a pk pair approach, storing the private key in GitHub Action secret and store the public key inside the QuickInstall repository.

Reply:
I think we should leave that off until later because there's a different conversation here in that quickinstall is a third party and signing might give a wrong impression that it's author-approved.

But yeah, if quickinstall starts signing its builds we can use a permanent keypair and hardcode the public key in binstall essentially.

Reply:
I can definitely do that since quickinstall is now part of cargo-bins.

Reply:
Opened a new issue there referencing your comment